web/src/routes/(user)/share/[key]/[[photos=photos]]/[[assetId=id]]/+page.svelte

```diff
@@ -80,8 +81,10 @@
         {$t('sharing_enter_password')}
       </div>
       <div class="mt-4">
-        <form novalidate autocomplete="off" {onsubmit}>
-          <input type="password" class="immich-form-input mr-2" placeholder={$t('password')} bind:value={password} />
+        <form class="flex" novalidate autocomplete="off" {onsubmit}>
+          <div class="relative mr-2">
```

Comment:
I don't think you need the changes here either, right? Get rid of the `class="flex"` on the form and remove the div again.

Reply:
The `class="flex"` on the form is to place the password input and the submit button on the same row per the original design. The `PasswordField` component is a div so it pushes the button into the next row if the form is not a flexbox.

It looks like I can get rid of `<div class="relative mr-2">` that wraps the password field and move the margin to onto the button `<Button class="ml-2" ...>`.

Reply:
I know what that flex is. I just didn't see how swapping the input with the password component would make the difference here. If it's required though we obv can keep it, thus the question :)
Instead of ml/mr I'd rather work with `gap` I think.